app/views/appointment_summaries/email_confirmation.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="page-header">
+  <h1>Make sure this is right</h1>
+</div>
+
+
+<%= form_for @appointment_summary,
+             url: preview_appointment_summaries_path,
+             method: :post do |f| %>
+  <%= error_messages_for(@appointment_summary) %>
+
+  <% (AppointmentSummary.columns.map(&:name) - %w(id created_at updated_at user_id email)).each do |field| %>
+    <%= f.hidden_field field %>
+  <% end %>
+
+  <div class="form-group">
+    <%= f.label :email %>
+    <%= f.text_field :email, class: %w(form-control t-email input-md-3) %>
+  </div>
+
+  <%= f.submit %(Yes - this is the customer's email address),
+               class: %w(btn btn-lg btn-success t-confirm),
```

Comment:
it is the apostrophe that makes it not a string.

And used % notation over " because of the reasons..... will change it if you prefer.